packages/nuxt/src/app/composables/payload.ts

```diff
@@ -19,34 +19,22 @@ interface LoadPayloadOptions {
 /** @since 3.0.0 */
 export async function loadPayload (url: string, opts: LoadPayloadOptions = {}): Promise<Record<string, any> | null> {
   if (import.meta.server || !payloadExtraction) { return null }
-  const payloadURL = await _getPayloadURL(url, opts)
-  const nuxtApp = useNuxtApp()
-  const cache = nuxtApp._payloadCache ||= {}
-  if (payloadURL in cache) {
-    return cache[payloadURL] || null
+  // TODO: allow payload extraction for non-prerendered URLs
```

Comment:
Will this be part of a separate PR?

Reply:
yes.

it's an existing todo that i forgot to write in previously (ie it's not changed by this pr)